src/views/basket.html

```diff
@@ -18,9 +18,10 @@ <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title
   </div>
   <div class="govuk-notification-banner__content">
     <p class="govuk-notification-banner__heading">
-      Your basket is full. You cannot add more than {{ basketLimit }} items to your order.
-      Remove some items before adding more.
+      Your basket is full. 
```

Comment:
Do we want the full stop at the end of this heading?

Reply:
It is like that in the image attached to the ticket